### DISCUSSION TITLE
Issue with SD Automatic 1111 The Last Ben running on Google Colab

### DISCUSSION THREAD

**Discussion by HappyRobotLtd** [TheLastBen/fast-stable-diffusion#2742] [General]
Hi,

I'm running The Last Ben/ SD Automatic 1111 via Google Colab and recently installed AnimateDiff.

After updating it each time to the latest version (1.6.1?) I have noticed that the Motion Modules required by AnimateDiff are no longer visible, making it effectively redundant. Reloading the UI / restarting Automatic 1111 makes no difference to the current state.

They did show up when I installed AnimateDiff a matter of a week or so ago, but now no longer.

Any suggestions as to why this might be ?

R

**Comment by ClaraSanders** [comment]
Sorry, I don't use AnimateDiff. Vanilla Automatic works for me.

**Comment by TheLastBen** [comment]
what is the full error log ?

**Comment by HappyRobotLtd** [comment]
Hi,

No error log as such - the motion modules simply don’t show up in the drop-down at all and therefore I can’t run AnimateDiff.

I’ve checked to make sure they are in the correct folder and they are.

I tried deleting some of the modules to leave only one, thinking it may be a memory issue connected to a large a custom style checkpoint, but that makes no difference.

R

**Comment by HappyRobotLtd** [comment]
*I’m running it via Google Colab, not a local install.

I’ve noticed SD has now updated to version 1.7.1 and no change re AnimateDiff motion modules.

R

**Comment by HappyRobotLtd** [comment]
Hi,

Issue solved…Duh…I checked the settings for AnimateDiff and the location indicated for the motion modules was wrong, or rather was a different location to the expected one.

I have now moved the modules to their new location and they show up fine in the drop-down.

R

**Comment by HappyRobotLtd** [comment]
Hi,

I’m finding it impossible to run Automatic 1111 in Google Colab due to errors and also Deforum has vanished from the tabs even though it says it’s installed.

No amount of restarting the UI makes any difference - quiet worrying since I thought i was making good progress with Deforum only yesterday.

Any ideas why ?

R

**Reply by ClaraSanders** [reply]
I think this is due the new transformers. The author of Deforum needs to update the add-on. It happened to me with Super Merger. I restarted and reinstalled the add-on again and again. And then yesterday I updated to the latest version and it works now.
